cmd/quota.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	k8sResource "k8s.io/apimachinery/pkg/api/resource"
+)
+
+var quotaConfig quotaFlags
+
+type quotaFlags struct {
```

Comment:
Let's move the variables into https://github.com/Shopify/kubeaudit/blob/master/cmd/root.go we decided to keep all the variables there. And did so for all the other commands.